src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -66,39 +52,8 @@ void main()
     Normal = normalMat*normalize(vNormal);
 }
 )";
-    constexpr const char* FRAG_SHADER_NO_UV =
-R"(#version 430 core
-
-layout (location = 0) in vec3 LocalPos;
-layout (location = 1) in vec3 ViewPos;
-layout (location = 2) in vec3 Normal;
-layout (location = 0) out vec4 OutColor;
-
-#define ILLUM_MODEL_MASK 0x0fu
-layout (push_constant) uniform Block {
-    vec3 Ka;
-    vec3 Kd;
-    vec3 Ks;
-    vec3 Ke;
-    vec4 Tf;//w component doesnt matter
-    float Ns;
-    float d;
-    float bm;
-    float Ni;
-    float roughness;
-    float metallic;
-    float sheen;
-    float clearcoatThickness;
-    float clearcoatRoughness;
-    float anisotropy;
-    float anisoRotation;
-    //extra info
-    uint extra;
-} PC;
-
-#include <irr/builtin/glsl/brdf/specular/fresnel/fresnel.glsl>
-
-#define Ia 0.1
+    constexpr const char* FRAG_SHADER_NO_UV_MAIN =
+R"(#define Ia 0.1
 void main()
 {
     vec3 N = normalize(Normal);
```

Comment:
still need to use the a common brdf util header to get all the vectors (that common code I linked you).